MANIFEST.in

```diff
@@ -3,7 +3,7 @@ include CONTRIBUTING.rst
 include HISTORY.rst
 include LICENSE
 include README.rst
-recursive-include cid *.html *.png *.gif *js *.css *jpg *jpeg *svg *py
+recursive-include src/cogite *.html *.png *.gif *js *.css *jpg *jpeg *svg *py
```

Comment:
pretty sure `cogite` isn't intended :)

Also, shouldn't there be a modification in `setup.py` to support the move to `src/`?

Reply:
I did not copy and paste this line from another project. No. Not at all. I would never do that.

> Shouldn't there be a modification [...] to support the move to src/?

Good catch. I did that in [another commit](https://github.com/Polyconseil/django-cid/pull/62/commits/af730d708dcb70b0adcc04749371a79eaba0ec04) later. So... this commit is not atomic! I'll probably go to hell for that.